app/models/pin.rb

```diff
@@ -1,6 +1,5 @@
 class Pin < ApplicationRecord
-
   belongs_to :creator, class_name: 'User'
   has_many :comments
-  has_many :pintags
+  has_many :pin_tags
```

Comment:
need clarification between pintags and pin_tags. This was changed to pin_tags, however, in the PinTagVote model it is referenced as pintag.  Is this intentional?